newIDE/app/src/MainFrame/Providers.js

```diff
@@ -34,6 +34,7 @@ import { GamesShowcaseStateProvider } from '../GamesShowcase/GamesShowcaseContex
 import { TutorialStateProvider } from '../Tutorial/TutorialContext';
 import ConfirmProvider from '../UI/Confirm/ConfirmProvider';
 import { AnnouncementsFeedStateProvider } from '../AnnouncementsFeed/AnnouncementsFeedContext';
+import PrivateAssetsProvider from '../AssetStore/PrivateAssets/PrivateAssetsAuthorizationProvider';
```

Comment:
The name of the default import should be the name of the file. Change either one or the other according to what makes sense.